### PR TITLE
Trend features

### PR DESCRIPTION
**Pull request by MaxBenChrist**
I added a feature that captures characteristics about the overall trend. 

It fits a linear regression model and returns characteristics of it such as slope or intercept.

### CODE REVIEW

File: tests/feature_extraction/test_feature_calculations.py [hunk 1]
```diff
@@ -568,22 +575,52 @@ def test_friedrich_coefficients(self):
...
+        self.assertTrue(abs(ds.deterministic - v0) < 0.0001)
 
         # Brownian motion
-        ds = velocity(tau=2.0/0.3-3.8, delta_t=0.05, R=3e-4, seed=0)
+        ds = velocity(tau=2.0 / 0.3 - 3.8, delta_t=0.05, R=3e-4, seed=0)
         v = ds.simulate(1000000, v0=np.zeros(1))
-        v0 = max_langevin_fixed_point(v[:,0], **default_params)
-        self.assertTrue(v0<0.001)
+        v0 = max_langevin_fixed_point(v[:, 0], **default_params)
+        self.assertTrue(v0 < 0.001)
```
Comment (nils-braun) [inline tests/feature_extraction/test_feature_calculations.py]: Yes, I know. You wanted to adjust the overall code style.... But, as I always say, please use another PR for this :-P ;-)

Comment (MaxBenChrist) [inline tests/feature_extraction/test_feature_calculations.py]: ok. next time ;)

Comment (nils-braun) [inline tests/feature_extraction/test_feature_calculations.py]: Thanks for reading my review... :-P

Comment (MaxBenChrist) [inline tests/feature_extraction/test_feature_calculations.py]: ;)

**Pull request merged**